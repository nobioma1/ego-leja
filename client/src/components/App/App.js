import { CSSReset, Box } from '@chakra-ui/core';
import React from 'react';

import { ThemeProvider } from 'components/Theme';
import { AppRoute } from 'components/Routes';

export const App = () => {
  return (
    <ThemeProvider>
      <CSSReset />
      <Box>
        <AppRoute />
      </Box>
    </ThemeProvider>
  );
};
