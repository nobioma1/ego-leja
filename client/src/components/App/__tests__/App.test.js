import React from 'react';
import { render, cleanup } from '@testing-library/react';

import { App } from '../App';

afterEach(cleanup);

describe('App Component', () => {
  it('Match snapshot', () => {
    const { asFragment } = render(<App />);

    expect(asFragment(<App />)).toMatchSnapshot();
  });
});
