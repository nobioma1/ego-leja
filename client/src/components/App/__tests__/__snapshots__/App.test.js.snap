// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component App Component match snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <div
      class="css-b4i9nx"
    >
      <div
        class="css-1y5qxzn"
      >
        <div
          class="css-z7mtfw"
        >
          <img
            alt="Credit logo"
            class="css-1kldd4v"
          />
          <div
            class="css-1xhj18k"
          >
            <button
              class="css-s1jo4v"
              type="button"
            >
              Login
            </button>
            <button
              class="css-6cthfa"
              type="button"
            >
              Create An Account
            </button>
          </div>
        </div>
        <div
          class="css-qv7yi4"
        >
          <div
            class="css-a2hsxd"
          >
            <h2
              class="css-19put2e"
            >
              Credit tracking, 
              <br />
              has never been so simplified.
            </h2>
          </div>
          <p
            class="css-1jog7l0"
          >
            A money lending management web application that will keep track of your lendings, payments, notify on due dates, pending amount...
          </p>
        </div>
      </div>
      <div
        class="css-1t1bi30"
      />
    </div>
  </div>
</DocumentFragment>
`;
